docs/lib/storage/fragments/js/copy.md

```diff
@@ -0,0 +1,116 @@
+## Copy
```

Comment:
remove H2 because page header is already H2. this would render as: 
"Copy files"
"Copy"

which would look weird.